README.md

```diff
@@ -54,7 +54,7 @@ Server via the container.
 
 ### Requirements
 
-- `jupyterlab>=2` or `notebook`
+Either `jupyterlab>=3` or `notebook<7` is required.
```

Comment:
Shouldn't notebook>=7 also work, since it's based on JupyterLab?

Reply:
Maybe hmm maybe, I just saw a lot of tests against notebook<7 specifically as compared to any version of notebook.

Should i adjust this to be as it was, without the <7 part?